exercises/connect/connect_test.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+import connect
+
```

Comment:
Could you please also leave a comment stating what version of `canonical-data.json` the tests were adopted as discussed in #784 if aplicable?
The format is:
```text
# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0
```

Reply:
updated!